packages/extensionmanager/src/widget.tsx

```diff
@@ -174,37 +160,51 @@ function ListEntry(props: ListEntry.IProperties): React.ReactElement<any> {
           {entry.description}
         </div>
         <div className="jp-extensionmanager-entry-buttons">
-          <button
-            className="jp-extensionmanager-install"
-            onClick={() => props.performAction('install', entry)}
-          >
-            Install
-          </button>
-          <button
-            className="jp-extensionmanager-update"
-            // An install action will update the extension:
-            onClick={() => props.performAction('install', entry)}
-          >
-            Update
-          </button>
-          <button
-            className="jp-extensionmanager-uninstall"
-            onClick={() => props.performAction('uninstall', entry)}
-          >
-            Uninstall
-          </button>
-          <button
-            className="jp-extensionmanager-enable"
-            onClick={() => props.performAction('enable', entry)}
-          >
-            Enable
-          </button>
-          <button
-            className="jp-extensionmanager-disable"
-            onClick={() => props.performAction('disable', entry)}
-          >
-            Disable
-          </button>
+          {!entry.installed && (
```

Comment:
If I understand correctly, you are changing this to not render the element at all, instead of just hiding it with CSS. Seems like better to me.

I think you could remove the CSS rules for all of these now? Like `jp-extensionmanager-uninstall`?